virtualenv/_utils.py

```diff
@@ -1,25 +1,34 @@
 from __future__ import absolute_import, division, print_function
 
-import errno
 import os
+import re
 import shutil
 import stat
 
 
 def ensure_directory(directory, *args, **kwargs):
-    try:
+    # Fail if the destination exists and it's not a directory
```

Comment:
So no, this is not merely a style change. This is to bail early when due to some misconfiguration or bug the directory copying from the builder would attempt to copy a directory when there already a file with the same name in the destination.